components/query/default-info-panel-body.directive.ts

```diff
@@ -0,0 +1,14 @@
+import {Component, OnInit} from '@angular/core';
+import {HsQueryBaseService} from './query-base.service';
+
+@Component({
+  selector: 'hs-query-default-info-panel-body',
+  template: require('./partials/default-info-panel-body.html'),
+})
+export class HsQueryDefaultInfoPanelBodyComponent implements OnInit {
```

Comment:
What about renaming this file to `default-info-panel-body.component.ts` ?

Reply:
Can do